polyply/src/topology.py

```diff
@@ -333,7 +333,7 @@ def add_positions_from_file(self, path, build_res=[]):
 
             for node in meta_mol:
                 resid = meta_mol.nodes[node]["resid"]
-                atoms_in_res = find_atoms(meta_mol.molecule, "resid", resid)
+                atoms_in_res = list(meta_mol.nodes[node]["graph"].nodes)  #find_atoms(meta_mol.molecule, "resid", resid)
```

Comment:
???

Reply:
it's a residue graph, so we don't actually have to find the atoms based on resid anymore.